tests/model/test_model.py

```diff
@@ -1585,7 +1585,7 @@ def test_is_muted_stream(self, muted_streams, stream_id, is_muted,
     ])
     def test_is_muted_topic(self, topic, is_muted, stream_dict, model):
         model.stream_dict = stream_dict
-        model.muted_topics = [
+        model._muted_topics = [
```

Comment:
Hmm. I am unable to find any other `muted_topics` instance in the code. Could you point out where?

Re commit, I am inclined towards having it here since we replace the old references with `is_muted_topic()` and it is easier to follow with less number of changes.

Reply:
I can't find another instance now, so this looks good :+1: